static/app/gettingStartedDocs/python/django.tsx

```diff
@@ -76,21 +89,36 @@ ${sentryInitContent}
       {
         language: 'python',
 
-        code: `
+        code: `# urls.py
 from django.urls import path
 
 def trigger_error(request):
     division_by_zero = 1 / 0
 
-    urlpatterns = [
-        path('sentry-debug/', trigger_error),
-        # ...
-    ]
+urlpatterns = [
+    path('sentry-debug/', trigger_error),
+    # ...
+]
         `,
       },
     ],
-    additionalInfo: t(
-      'Visiting this route will trigger an error that will be captured by Sentry.'
+    additionalInfo: (
+      <span>
+        <p>
```

Comment:
It works but the HTML is not valid. `span` is an inline element whereas `p` is a block element. Block elements may not be nested inside inline elements. Better use a `div` instead of the `span` here.